src/Core/Resolvers/SqlQueryEngine.cs

```diff
@@ -208,6 +210,40 @@ private OkObjectResult FormatFindResult(JsonDocument jsonDoc, FindRequestContext
             // nextLink is the URL needed to get the next page of records using the same query options
             // with $after base64 encoded for opaqueness
             string path = UriHelper.GetEncodedUrl(_httpContextAccessor.HttpContext!.Request).Split('?')[0];
+
+            RuntimeConfig runtimeConfig = _runtimeConfigProvider.GetConfig();
+            // If the base route is not empty, we need to insert it into the URI before the rest path.
```

Comment:
Instead of inserting the base-route with substring etc, can we construct the full path ourselves since we know all the components anyway? 

E.g. Can we use this: https://devio.wordpress.com/2009/10/19/get-absolut-url-of-asp-net-application/ ? 

What will be more efficient and accurate to do?

Can we move this logic into a small helper function?

Reply:
With Sean's suggestion, this is taken care of.